wordpress-mobile/WordPress-iOS
# Create Header

Fixes #13717

This PR adds a Header/Navigation bar at the top of the Nudges view.

### To test

1. Tap Publish
2. Check if the information about the blog are correct
3. Tap "Tags" and check that the header now shows a back button
4. Tap the back button (or use the gesture to dismiss the ViewController)

Please, test on the iPad and iPhone.

PR submission checklist:
